apps/prairielearn/src/pages/instructorAssessmentManualGrading/assessmentQuestion/utils/columnDefinitions.tsx

```diff
@@ -0,0 +1,420 @@
+              );
+              return (
+                <span class="d-flex align-items-center gap-2">
+                  {value}
+                  {group && (
+                    <OverlayTrigger
+                      overlay={<Tooltip>{group.instance_question_group_description}</Tooltip>}
+                    >
+                      <span>
+                        <i class="fas fa-circle-info text-secondary" />
+                      </span>
+                    </OverlayTrigger>
+                  )}
+                </span>
+              );
+            },
+            filterFn: (row, columnId, filterValues: string[]) => {
+              if (filterValues.length === 0) return true;
+              const current =
+                row.getValue<InstanceQuestionRow['instance_question_group_name']>(columnId);
+              if (!current) {
+                return filterValues.includes('No Group');
+              }
+              return filterValues.includes(current);
+            },
+          }),
+        ]
+      : []),
+
+    // User/Group name column (hidden by default)
+    columnHelper.accessor('user_or_group_name', {
+      id: 'user_or_group_name',
+      header: groupWork ? 'Group Name' : 'Name',
+      cell: (info) => info.getValue() || '—',
+      enableHiding: true,
+    }),
+
+    // UID column (hidden by default)
+    columnHelper.accessor('uid', {
+      id: 'uid',
+      header: groupWork ? 'UIDs' : 'UID',
+      cell: (info) => {
+        const uid = info.getValue();
+        const enrollmentId = info.row.original.enrollment_id;
+        if (!uid) return '—';
+        if (enrollmentId) {
+          return <a href={getStudentEnrollmentUrl(urlPrefix, enrollmentId)}>{uid}</a>;
+        }
+        return uid;
+      },
+      enableHiding: true,
+    }),
+
+    // Grading status column
+    columnHelper.accessor('requires_manual_grading', {
+      id: 'requires_manual_grading',
+      header: 'Grading status',
+      cell: (info) => (info.getValue() ? 'Requires grading' : 'Graded'),
+      filterFn: ({ getValue }, columnId, filterValues: string[]) => {
+        if (filterValues.length === 0) return true;
+        const requiresGrading = getValue(columnId);
+        const status = requiresGrading ? 'Requires grading' : 'Graded';
+        return filterValues.includes(status);
+      },
+    }),
+
+    // Assigned grader column
+    columnHelper.accessor('assigned_grader_name', {
+      id: 'assigned_grader_name',
+      header: 'Assigned grader',
+      cell: (info) => info.getValue() || 'Unassigned',
+      filterFn: (row, columnId, filterValues: string[]) => {
+        if (filterValues.length === 0) return true;
+        const current = row.getValue<InstanceQuestionRow['assigned_grader_name']>(columnId);
+        if (!current) return filterValues.includes('Unassigned');
+        return filterValues.includes(current);
+      },
+    }),
+
+    // Auto points column (only if assessment has auto points)
```

Comment:
Points like this are a property of assessment questions, not assessments:

```suggestion
    // Auto points column (only if assessment question has auto points)
```